### ISSUE TITLE
duplicating a channel leads to an error

### ISSUE THREAD

**Issue by fritzmg** [contao/newsletter-bundle#15]
Reproduction:

1. Log in to https://demo.contao.org/contao.
2. Go to _Newsletters_ and [edit the recipients](https://demo.contao.org/contao?do=newsletter&table=tl_newsletter_recipients&id=3) of _Newsletter - Demo_.
3. [Add](http://c44.local/contao?do=newsletter&table=tl_newsletter_recipients&id=1&act=create&mode=2&pid=1) `example1@example.org` and `example2@example.org`.
4. Go back to _Newsletters_ and [copy](https://demo.contao.org/contao?do=newsletter&act=copy&id=3) the _Newsletter - Demo_ channel.

You will see the following error:
```
An exception occurred while executing 'INSERT INTO tl_newsletter_recipients (`pid`, `tstamp`, `email`, `active`, `addedOn`, `confirmed`, `ip`, `token`) VALUES ('7', 1521189960, '', '', '', '', '', '')': SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '7-' for key 'pid_email'
```
This happens because during the duplication the email address for each duplicated recipient is removed.

**Comment by leofeyer** [comment]
Does this affect Contao 4.4 as well?

**Comment by fritzmg** [comment]
Yes, Contao 4.4.16 and 4.5.6 are affected.

**Comment by leofeyer** [comment]
Does it even make sense to duplicate newsletter recipients when a channel is duplicated?

**Comment by rflx** [comment]
@leofeyer you mean to only duplicate the newsletter channel settings without its recipients? I think it makes sense to duplicate the recipients, otherwise you'll need to export/import the recipients from the main channel if you want to copy them to a new one.

**Comment by leofeyer** [comment]
I guess you are right, although I'm not sure if exporting and importing (a.k.a. subscribing a recipient to a new channel without double opt-in) is still allowed under the regulations of the GDPR. But we will know soon. 😄

**Comment by leofeyer** [comment]
The issue is caused by the `'doNotCopy'=>true` flag which we have added in a2151410d228f40aa18b5bf3231bfedc972532e5. If the flag is removed from the `email` and `active` fields, the recipients and their activation status are copied.

The flag is not set in Contao 3.5, therefore I think we can remove it in Contao 4.4 again. However, the activation status is not copied neither in Contao 3.5 nor in Contao 4, which means all duplicated recipients are disabled by default. This is not what we want, is it?

@contao/developers /cc

**Comment by aschempp** [comment]
I don't understand. How does it make sense to duplicate a recipient but not copy it's email address???

**Comment by frontendschlampe** [comment]
I think, it's inadmissible to copy a channel with it's mail addresses because of privacy. If a user subscribed for a special channel with special information, than he want to have only these information and not any other one ... so we have to deactivate the copy function for channels.

**Comment by rflx** [comment]
@frontendschlampe there is always a way to duplicate them if you want to. I think Contao should give the admin the possibility to duplicate them (not for abuse purposes). Privacy is very important but i think in this case its not an issue of Contao.

**Comment by frontendschlampe** [comment]
Yes, but we also add the ```'doNotCopy'=>true``` for a single recipient ... if anybody find another way to copy them, they should do, but we should not support them.

**Comment by aschempp** [comment]
> If a user subscribed for a special channel with special information, than he want to have only these information and not any other one

They don't necessary sign up for a special channel, they might sign up for your news. Noone knows people organize their newsletters and channels. Imho they should not be marked as double-opt-in but simply as manually added, but they sure should be copied.

**Comment by aschempp** [comment]
Also, not copying the email does not do what you describe. It **does** copy all recipient records but empties the email field. Which means you get 1000 subscribers without email. Certainly not useful.

**Comment by leofeyer** [comment]
Fixed in 04ad925daa1bdf51522dabb13acfcbe468c5c4eb.
